### PR TITLE
feat: Success notification unlockable drop

### LINKED ISSUE #6204: Success notification unlockable drop
image
### main
- the thing will be shown when you click the mint button 
- there is a copy saying that the nft will be visible in an estimated 30 seconds 
- once the 30 seconds pass, the button with 3 dots animation and copy "preparing your item..." will change to "view item" 
- button redirects the user to the nft detail. 
- there is also an option to close this notification 
- should pop up in the middle of the screen - with darkened background (same as with the sidebar) 
- clicking outside the modal should not close the notification since I personally always accidentally close it :D 
- handoff file is updated - page unlockable drop page: https://www.figma.com/file/JXNtryaNYhY0XJNArzvPQx/Handoff?type=design&node-id=589%3A6842&t=tGllyxKdl4QGJTHF-1

cc @vikiival I believe he has begun working on something?
- #6163

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot - One Stop Shop for Polkadot NFTs](https://kodadot.xyz).

👇 __ Let's make a quick check before the contribution.

## PR Type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

## Needs Design check

- @exezbcz please review

## Context

- [x] Closes #6204
- [ ] Requires deployment <snek/rubick/worker>

#### Did your issue had any of the "$" label on it?

- [x] My DOT address: [Payout](https://canary.kodadot.xyz/dot/transfer/?target=16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)

## Screenshot 📸

- [x] My fix has changed UI

image

image

### <samp>🤖 Generated by Copilot at 2f58564</samp>

Added a new `UnlockableLoader` component to improve the user experience when minting and viewing unlockable NFTs. Replaced the old `Loader` component with the new one in the `UnlockableContainer` component. Used some components and hooks from the `@kodadot1/brick` library.

### <samp>🤖 Generated by Copilot at 2f58564</samp>

> _To mint NFTs with a flair_
> _We added a new `UnlockableLoader` there_
> _It uses some hooks and props_
> _And some `Neo` components on top_
> _And shows a congrats message with care_

### CODE REVIEW

**Comment by Jarsen136** [comment]
> Great work! Is there a way to activate the popup without minting the NFT? Could you guide me to a resource where I can find more information on this? Thanks!

Yes, I have made a preview env for testing. Opening the page with parms `?redesign=true`, the popup would be active auto and then the button would be changed after a few seconds.
`https://deploy-preview-6213--koda-canary.netlify.app/stmn/unlockable?redesign=true`.

I will remove this testing logic before the PR is about to merge.

File: components/collection/unlockable/UnlockableLoader.vue
Comment (roiLeo) [inline components/collection/unlockable/UnlockableLoader.vue:78]: i18n

**Review by roiLeo — COMMENTED** [review]
Text look a bit small on figma

Why don't we unify component between these two

Capture d’écran 2023-06-14 à 8 41 39 AM
Capture d’écran 2023-06-14 à 8 41 29 AM

**Comment by exezbcz** [comment]
@roiLeo its a different success, the one for drop page is in figma drop page

File: components/collection/unlockable/UnlockableLoader.vue
Comment (vikiival) [inline components/collection/unlockable/UnlockableLoader.vue:20]: Isn't there some nicer way how to load svgs?

Comment (Jarsen136) [reply components/collection/unlockable/UnlockableLoader.vue]: Yes, there is

**Review by roiLeo — APPROVED** [review]
in anticipation of a unified design, let's roll

**Comment by exezbcz** [comment]
okay, a few minor changes 
- make the background the same as with opening the wallet connect sidebar - if its already, my bad :D 
- The button should be 16px font size 
- make the box close only with the close icon because otherwise, it could happen you accidentally close it. 

Thanks! 

image

**Review by vikiival — APPROVED** [review]

**Comment by exezbcz** [comment]
> It's already there.
> 
> >

Cool, I think I will create one more issue for some small things, I don't want to hold up this one. Thanks! 

visual oki 🏁

**Comment by yangwao** [comment]
pay 30 usd

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $30 @ 4.74 USD/DOT ~ 6.329 $DOT
🧗 [16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z](https://polkadot.subscan.io/account/16SjUbGKSdjCdWTy3NNT3JxbRVGGqD4mwkHpc6BD9U2Rp29Z)
🔗 [0x84c23fa2b4fd20e12a49840de308b4091c5f695e6dfe94c0cd65a186e633b6a6](https://polkadot.subscan.io/extrinsic/0x84c23fa2b4fd20e12a49840de308b4091c5f695e6dfe94c0cd65a186e633b6a6)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**
